src/devices/BrickPi3/README.md

```diff
@@ -311,3 +311,47 @@ The only supported sensors in the GrovePi port present on BrickPi3 are I2C senso
 
 A series of hardware tests for motors and sensors are available in [BrickPi3.samples](./samples). Those hardware tests offers a variety of low level access to the Brick class as well as high level thru the Motor and Sensor classes.
 
+## Example
+
+Refer to the [sample code](./samples) to understand on which port you'll need to plug motors and sensors. The available tests are the following:
```

Comment:
Suggesting to remove links to samples - this will not look good if we port these docs to the official documentation - most likely the link will be broken

Reply:
That's a good catch. We may then want to replace by an absolute link on the repo?